buddypress/members/single/edit.php

```diff
@@ -595,12 +595,15 @@
             </label>
         </section>
         -->
-        <?php if(!isset($meta['agree'][0]) || $meta['agree'][0] != 'I Agree'): ?>
-        <?php if(sizeof($guidelines) === 1): ?>
+        <!-- <?php if(!isset($meta['agree'][0]) || $meta['agree'][0] != 'I Agree'): ?> -->
```

Comment:
This shouldn't be commented out.  I assume it was commented out for testing purposes?